### DISCUSSION TITLE
Who's hiring (Oct 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#41663] [Ideas]
# What is this?
New month, new post! This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #41662.

*If you're looking for last month's discussion, see #40173. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!*

**Comment by gabrielbull** [comment]
Pool Office Manager | Frontend Engineer | REMOVE | Full-time | $40-65/h | gabriel@poolofficemanager.com

We are a SaaS company building a tool to manage/schedule/invoice for pool service companies (think pool cleaners). We are a company based in Columbus, Ohio, here is our website: [https://poolofficemanager.com/](https://poolofficemanager.com/)

We are looking for a talented React developers, to work on the this stack: 

- React
- Typescript
- NextJS
- GraphQL
- CSS/Styled-Components

Please drop me an email at gabriel@poolofficemanager.com

**Reply by zhyd1997** [reply]
Hi 👋

Please take a look my portfolio:
https://github.com/vercel/next.js/discussions/41662#discussioncomment-3941891

**Reply by vigneshshiv** [reply]
Hello, Please checkout my portfolio
[#41662 (comment)](https://github.com/vercel/next.js/discussions/41662#discussioncomment-3972655)

**Reply by Cheetah0723** [reply]
Hello, thank you for job posting.
I am interested in this position.
https://iyuki.info

**Reply by vishal590** [reply]
Check out my portfolio 
https://portfolio.vishallambe.repl.co/

**Comment by demc** [comment]
Niche Protocol, Inc | Senior Software Engineer | NYC / LA | Full-time | $140-170k base | https://niche.club

Niche is a new ad-free web3 social media platform that aims to give the benefits of creating amazing content back to content creators and their audiences. Founded by Christopher Gulczynski (the creator of the dating apps Tinder and Bumble) and Zaven Nahapetyan (long-time Facebook engineering leader), Niche is funded by top-tier investors, including Sequoia Capital.

We are leveraging cutting-edge decentralized technologies to enable online creators (YouTubers, Instagrammers, TikTokers, etc) to create and monetize their audience through private communities. Creators and followers post and converse about mutual interests and passions in ad-free clubs. We aim to reinforce throughout our communities the future of social media: interest-based connections and valuable real-life relationships.

Read about us in the news!

https://www.protocol.com/entertainment/bumble-facebook-niche-web3
https://www.wundermanthompson.com/insight/niche-q-a-co-founders-christopher-gulczynski-ceo-and-zaven-nahapetyan-cto
https://uk.news.yahoo.com/facebook-web-3-challengers-revolutionise-social-media-083518898.html?guccounter=1

We are seeking a senior software engineer to join our intimate engineering team.

**Skills (3+ years)**

TypeScript/JavaScript for web and mobile
Database schema design and implementation
NodeJS API design
Unit testing / dependency injection with Jest (or similar)

**Responsibilities**

Build mobile applications using React Native.
Design database schemas that can scale to millions of users.
Architect server-less backend infra using Google Cloud Functions or AWS Lambda Functions.
Contribute meaningfully to code review and technical decision making.


Please contact the hiring manager:

Derrick McMillen
derrick@niche.club

**Reply by Cheetah0723** [reply]
Hello, thank you for job posting.
I am interested in this position.
https://iyuki.info/

**Comment by Yunesss** [comment]
SUBUP | FRONTEND | SEOUL, KOREA | Full-time | subup.io

We started based on a project to advance the video translation service in South Korea.  Our mission is to offer an affordable service to Korean content creators to bring them closer to their overseas audiences and global content creators to their korean audiences . On our journey for that we built a web front to automate the orders. Now we are working on a web subtitle editor that supports voice recognition AI, speech-to-text AI  and polished by humans.

We are looking for talents that are familiar with this tech stack :

- React
- NextJS
- TypeScript
- Websockets (SocketIO)
- GraphQL
- CSS/Tailwind
- Docker
- CI/CD
- Some basic AI knowledge

Don't hesitate to send me an e-mail :  youness@yousub.io

**Reply by Anishpras** [reply]
@Yunesss I am interested in the job. You can check my GitHub - https://github.com/Anishpras and LinkedIn - https://www.linkedin.com/in/anishpras118/

**Reply by AsherAshrafM** [reply]
@Yunesss I am interested. Please see my portfolio. 
https://sachu-developer.webflow.io/
Thank you.

**Reply by Cheetah0723** [reply]
Hello, thank you for job posting.
I am interested in this position.
https://iyuki.info/

**Comment by voluntadpear** [comment]
PSPDFKit GmbH | Senior Frontend Developer | Worldwide remote | Full-time | https://pspdfkit.bamboohr.com/careers/85

PSPDFKit is the leading document processing and manipulation platform for developers and enterprise businesses. Our customers use our offering for creating, editing, signing, and collaborating on documents in their applications across mobile, Web, and all major platforms. Nearly a billion end-users in over 150 different countries rely on PSPDFKit every day including industry leaders like Dropbox, DocuSign, SAP, IBM, Volkswagen, Fabasoft, Wolters Kluwer Deutschland, and the European Patent Office.

**What you will do...**

- Contribute to the development of products used by millions of people every day
- Build new features and fix existing issues on a regular schedule
- Develop UI and UX according to specifications and design concepts
- Write readable and well-documented code
- Collaborate with a company distributed in more than 20 countries using Slack, GitHub, and Zoom as main communication and collaboration tools
- Manage your work by setting your own goals, prioritizing and executing them, and ultimately taking ownership of tasks and projects
- Innovate using the latest front-end technologies, frameworks, and approaches.


**About You...**

- Be familiar with JavaScript and its ecosystem of tools and packages (some of the tools we use are React, TypeScript, webpack, and WebAssembly)
- Be familiar with the web technology stack (e.g. HTTP, WebSockets, cookies, DOM, CSS, and the browser as a platform)
- Have a breadth-first approach to software engineering, looking outside the Web platform when needed (e.g. command-line tools, theorem provers, or other programming languages) to learn what’s needed to do your best work
- Have a strong interest in designing and maintaining a set of cohesive public APIs, always striving for simplicity and composability
- Have good communication skills and enjoy working with a passionate team, while understanding the challenges of working on a distributed team
- Have a well-rounded approach to problem solving, and understand the difference between when to apply a fix and when to refactor to remove a specific class of bugs
- Understand the value of changes in the larger context of the business and the product’s customer
- Have a good command of English for writing documentation and occasional blog posts, and provide direct support to our customers

**Reply by a-belard** [reply]
Hello, have a look at my portfolio :) https://angebelard.me

**Reply by voluntadpear** [reply]
Great portfolio @a-belard. Could you fill this form please if you're interested in the position? https://pspdfkit.bamboohr.com/careers/85. We would like to talk with you!

**Reply by a-belard** [reply]
Thanks, @voluntadpear. My application was submitted successfully.

**Reply by Cheetah0723** [reply]
Hello, thank you for job posting.
I am interested in this position.
https://iyuki.info/

**Comment by sunny-thunnus** [comment]
**LLGM | Sr Full Stack Developer/Architect | Hybrid (On-Site 2/3 days a week; Palm Beach, FL) | Full time | sunny@liquiditylock.com**

- Our team has been fortunate to have had two FinTech exits and would like to keep this resource in house to grow with us. The ideal candidate would like a wide range of roles and responsibilities and enjoys the pace of startups as we grow and integrate client demand and suggestions.
- Looking for someone who is an expert in UI design, UX design, and front-end development to join our growing company.
- You will be a natural at understanding human-computer interaction design (HCID), in coming up with elegant solutions to help design and develop our intuitive and revolutionary portfolio trading technologies.

_**Qualifications**_

- Must be authorized to work in the U.S
- 5+ years of experience with software development, APIs, and microservices
- 5+ years of experience with building software using agile methods
- 5+ years of experience in JavaScript frameworks (e.g. React, Node.js, Next.js, Express.js) 
- Proficient in CSS, HTML, and writing cross-browser compatible code
- Experience with SQL/NoSQL/NewSQL databases
- Familiarity with RESTful APIs and GraphQL
- Familiarity with modern front-end build pipelines and tools
- Familiarity with code versioning tools like git.
- A true colleague: Someone who understands a great client tells you what they don't like, a great employee tells you why and a true colleague presents different ways of remedying it.
- Must have a great design aesthetic and be prepared to walk us through your portfolio when you interview.

**Comment by SuperStar518** [comment]
Hello.
Going through your project description I found it to be the right fit for us.

I can do such an app from your posting and I will send you awesome results within your deadline if you hire me.
Actually, I am a Senior ReactJS Developer.
I have 5+ years’ experience working with JavaScript framework, preferably React/Redux.
I have a deep understanding of JavaScript , the DOM Elements and relevant concepts.
My primary focus will be on developing user interface components and implementing them following well-known React.js workflows (such as Flux or Redux and GraphQL or ApolloClient).
I will ensure that these components and the overall application are robust and easy to maintain.
I will try to commit to collaborative problem solving, guarantee sophisticated design, and quality product.
High coding quality guaranteed~
Also I am good at proficiency with browser debugging and performance testing and have good experience using Git and MVP frameworks including Jira.
I would like to inform you also that I am highly skilled with Node.js, Firebase, MongoDB, WebPack, ES6, API Integration, as I have a handsome experience over it.
I firmly believe that these qualities make me different from the rest of the contestants and also are exactly fit to the skills of your requirements
I am working on full-time job with freelancer and can work any time you want so feel free to keep in touch with you anywhere and whenever you want.
I would like to discuss more details in a chat or interview so we can get clear ideas with each other.
Thank you for taking time reviewing the letter.
I’ll look forward to hearing from you for further progress.

Best regards.

GitHub: https://github.com/SuperStar518
Stackoverflow: https://stackoverflow.com/users/6536335/superstar518

**Comment by Jared-Dahlke** [comment]
Personal | NextJS consultant | remote | one or 2 sessions | $100 per hour 

Hello,
We are converting our Create React App to NextJs but running into some issues related to authentication and appending JWT tokens to our headers. We have a standard homegrown auth system (UI calls /authenticate to get JWT token, and it appends that token onto every API call). I need someone to hop on a screenshare and walk me through what I am doing wrong. $100 per hour, or more if required. This should only take a couple hours at most if you know your stuff!

Right now i'm trying to wire it up using Iron-Session but not married to that.

**Reply by zhyd1997** [reply]
Any codebase available?

**Reply by Jared-Dahlke** [reply]
I’m not sure if I can post it online publicly but we could do a google meet ?

**Reply by zhyd1997** [reply]
To be honest, my English is not good, thanks for your time.

Maybe there will be a good candidate later.

**Reply by Anishpras** [reply]
Intrested

**Reply by Jared-Dahlke** [reply]
@Anishpras i just emailed you. What's the easiest way for me to share my screen with you so i can get some help? zoom or google meet?

**Reply by alexkvazos** [reply]
Happy to help if still available!

**Comment by cheeto-bandito** [comment]
[Merkle](https://www.merkle.com) | Sitecore Front-End Developer | US, Brazil, India - Remote | Full-time | $70k-105k base | [Job Req](https://www.merkle.com/careers/all-openings/743999857547808)

Merkle, a Vercel [partner](https://vercel.com/experts/merkle), is seeking a talented Sitecore Frontend Developer to contribute to a dynamic team building Sitecore-based web and native app solutions within our Experience Platforms group. This role will work with our development, creative, user experience, and project management teams to develop and deploy cutting-edge websites with innovative technologies.

**Reply by vishal590** [reply]
Check out my portfolio 
https://portfolio.vishallambe.repl.co/
